libs/features/mas/web-components/src/global.css.js

```diff
@@ -1073,6 +1084,11 @@ span[is="inline-price"][data-template='strikethrough'] {
     text-decoration: line-through;
 }
 
+sp-button a[is="checkout-link"] {
+  color: inherit;
+  text-decoration: none;
```

Comment:
I don't think this is needed since you already declared on L#613

Reply:
this file has grown too fast too big. We are going to split it.
merged `sp-button a` rules under the same rule.

Reply:
@3ch023 I change the rule as above, since merch-cards can have regular CTAs with links as well.